src/main/java/shiba/commands/HelpCommand.java

```diff
@@ -0,0 +1,128 @@
+package shiba.commands;
+
+import java.util.HashMap;
+
+import shiba.exceptions.ShibaException;
+import shiba.tasks.PersistentTaskList;
+import shiba.ui.Replier;
+
+/**
+ * Represents a command to display help messages.
+ */
+public class HelpCommand extends ShibaCommand {
```

Comment:
I think that it is nice that there is a "help" command that helps the user better understand the commands!